homeassistant/components/recorder/websocket_api.py

```diff
@@ -48,6 +48,7 @@
 
 UNIT_SCHEMA = vol.Schema(
     {
+        vol.Optional("conductivity"): vol.In(DataRateConverter.VALID_UNITS),
```

Comment:
This is incorrect and causes issues as noticed in a new issue. Uses wrong converter.

Reply:
Opened a PR to fix